### ISSUE TITLE
Spotless doesn't work with scalafmt 2.0.0

### ISSUE THREAD

**Issue by nsutcliffe** [diffplug/spotless#415]
I am using Spotless  version 3.23.1 with gradle.

I am using Spotless with Scalafmt, however if I try to specify version 2.0.0 (only released 8 days ago :-) ), this doesn't work as it looks like scalafmt has changed their package structure (no longer released under com.geirsson, but under org.scalameta. Spotless is till looking for it under com.giersson. Specific error below for reference:

```
Execution failed for task ':scoring-framework-model:spotlessScala'.
> Could not resolve all files for configuration ':detachedConfiguration2'.
   > Could not resolve com.geirsson:scalafmt-core_2.11:2.0.0.
     Required by:
         project :
      > Could not resolve com.geirsson:scalafmt-core_2.11:2.0.0.
         > Could not get resource
```

**Comment by nedtwigg** [comment]
Solved by #416, available in -SNAPSHOT

**Comment by nedtwigg** [comment]
Released in x.24.0
